cmd/kubeadm/app/util/version.go

```diff
@@ -251,7 +251,7 @@ func validateStableVersion(remoteVersion, clientVersion string) (string, error)
 	if verClient.Major() < verRemote.Major() ||
 		(verClient.Major() == verRemote.Major()) && verClient.Minor() < verRemote.Minor() {
 		estimatedRelease := fmt.Sprintf("stable-%d.%d", verClient.Major(), verClient.Minor())
-		klog.Infof("remote version is much newer: %s; falling back to: %s", remoteVersion, estimatedRelease)
+		klog.InfoS("Remote version is much newer, falling back", "remoteVersion", remoteVersion, "backToVersion", estimatedRelease)
```

Comment:
hi, kubeadm structured logging should not be done.

https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging#migration--graduation-criteria

> NOTE: The components kubectl and kubeadm are out of scope for this migration effort until further notice.